DATA/production/configurations/2022/LHC22f/apass1/setenv_extra.sh

```diff
@@ -172,18 +172,39 @@ else
   export ARGS_ALL_EXTRA=" --resources-monitoring 50 --resources-monitoring-dump-interval 50"
 fi
 
+#ALIGNLEVEL=0: before December 2022 alignment, 1: after December 2022 alignment
+ALIGNLEVEL=1
+if [[ $BEAMTYPE == "PbPb" || $PERIOD == "MAY" || $PERIOD == "JUN" || $PERIOD == "LHC22c" || $PERIOD == "LHC22d" || $PERIOD == "LHC22e" || $PERIOD == "LHC22f" ]]; then
+  ALIGNLEVEL=0
+fi
```

Comment:
I would use something like this, to consider the anchoring, assuming we will not anchor to anything smaller than apass2 for 13.6 TeV, and apass5 for PbPb:
```
if [[ $BEAMTYPE == "PbPb" || $PERIOD == "MAY" || $PERIOD == "JUN" || $PERIOD == "LHC22c" || $PERIOD == "LHC22d" || $PERIOD == "LHC22e" || $PERIOD == "LHC22f" ]]; then
    ALIGNLEVEL=0
    if [[ $ALIEN_JDL_LPMPRODUCTIONTYPE == "MC" ]]; then
	# extract pass number
	ANCHORED_PASS=$ALIEN_JDL_LPMANCHOREPASSNAME
	ANCHORED_PASS_NUMBER=`echo $ANCHORED_PASS | sed 's/^apass//'`
	echo ANCHORED_PASS_NUMER = $ANCHORED_PASS_NUMBER
	if [[ $PERIOD == "MAY" || $PERIOD == "JUN" ]] && [[ $ANCHORED_PASS_NUMBER -gt 1 ]]; then
	    ALIGNLEVEL=1
	elif [[ $PERIOD == "LHC22c" || $PERIOD == "LHC22d" || $PERIOD == "LHC22e" || $PERIOD == "LHC22f" ]] && [[ $ANCHORED_PASS_NUMBER -gt 2 ]]; then
	    ALIGNLEVEL=1
	fi
    fi
fi
```
Note that apass2 of 13.6 TeV will be done with a mixed configuration, though... This will be trickier, we need to go to the run numbers. 
@catalinristea , what do you think?